yes_no_maybe.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |spec|
     spec.metadata['allowed_push_host'] = "TODO: Set to 'http://mygemserver.com' to prevent pushes to rubygems.org, or delete to allow pushes to any server."
   end
 
-  spec.add_dependency 'activesupport', '~> 4.2', '>= 4.2.4'
+  spec.add_dependency "activesupport"
```

Comment:
I get not having a ceiling on the gem version if we're going to try and upgrade, but why not enforce a floor on the version?